decidim-budgets/db/migrate/20170612101846_migrate_projects_category.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class MigrateProjectsCategory < ActiveRecord::Migration[5.1]
+  def change
+    Decidim::Budgets::Project.find_each do |project|
+      Decidim::Categorization.create!(
+        decidim_category_id: project.category.id,
```

Comment:
Careful with this, as `project.category` has ceased to exist so this migration won't work. Again, I'd do this with pure SQL.